xml/depl_conf_admin_crowbar.xml

```diff
@@ -1483,6 +1482,11 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
    }
 }</screen>
 
+    <para>
+      The <literal>crowbar-<replaceable>FULLY QUALIFIED
+              HOSTNAME</replaceable></literal> scheme also applies to admin nodes, which are not booted from PXE. However, the details of the scheme are different than nodes which boot from PXE: there is no prefix d, and the &admserv; should have a <replaceable>admin.cloud.&exampledomain;</replaceable>-type FQDN, which is used in place of the MAC address. So the pattern in your conduit map looks like <literal>my_mode/.*/crowbar-admin_cloud_example_com_my_cloud</literal>
+</para>
+
```

Comment:
For the (there is only one) admin server and all nodes registered manually, a different naming scheme for the role name applies:

crowbar-&lt;replaceable>FQDN&lt;/replaceable>

where periods in  &lt;replaceable>FQDN&lt;/replaceable> are replaced by underscores

Consider moving this information up